ci/azure/windows.yml

```diff
@@ -11,49 +11,69 @@ jobs:
       py36_np15:
         ENV_FILE: ci/deps/azure-windows-36.yaml
         CONDA_PY: "36"
+        PATTERN: "not slow and not network"
 
       py37_np141:
         ENV_FILE: ci/deps/azure-windows-37.yaml
         CONDA_PY: "37"
+        PATTERN: "not slow and not network"
```

Comment:
Is this related? If not, I think we should have it in a separate PR.

Reply:
Okay agree - let me factor this out to a separate PR. This was part of the work to move the windows CI scripts to use bash.  (FYI I'm away from my keyboard for a few weeks - but will work on this when I return)

Reply:
can you factor this out